src/membraneWebRTC.ts

```diff
@@ -116,7 +116,10 @@ export interface TrackContext {
  * In other case, browser might not be able to scale
  * some encodings down.
  */
-export type TrackEncoding = "l" | "m" | "h";
+const TrackEncodingValues = ["l", "m", "h"] as const;
+export type TrackEncoding = typeof TrackEncodingValues[number];
+
+export const isTrackEncoding = (encoding: string): encoding is TrackEncoding => TrackEncodingValues.includes(encoding as TrackEncoding);
```

Comment:
It allows us to make strong type assertions. From `any` to `TrackEncoding`
<img width="891" alt="image" src="https://user-images.githubusercontent.com/19818474/198545183-e0149add-45ce-4d11-bf5d-38a5e98bbb9b.png">

<img width="429" alt="image" src="https://user-images.githubusercontent.com/19818474/198545287-c4f271cf-ed33-4caf-91b2-13497b1ab3c8.png">